docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -76,6 +76,18 @@ Noteworthy module changes
 * The ``bigip_device_facts`` module was renamed to :ref:`bigip_device_info <bigip_device_info_module>`.
 * The ``bigiq_device_facts`` module was renamed to :ref:`bigiq_device_info <bigiq_device_info_module>`.
 * The ``one_image_facts`` module was renamed to :ref:`one_image_info <one_image_info_module>`.
+- The ``digital_ocean_account_facts`` module was renamed to :ref:`digital_ocean_account_info <digital_ocean_account_info_module>`.
```

Comment:
Use `*` instead of `-` to continue this list.

Reply:
Nice catch.

Reply:
fixed